src/Assets/LightSkyDefense/Scripts/Enemies/EnemyAoEHeal.cs

```diff
@@ -0,0 +1,37 @@
+﻿using UnityEngine;
+
+public class EnemyAoEHeal : MonoBehaviour
+{
+    public float HealAmount = 0.1f;
+    public float HealInterval = 3;
+
+    // Start is called before the first frame update
+    private void Start()
+    {
+        AoEHealWithInterval();
+    }
+
+    private void AoEHealWithInterval()
+    {
+        HealEnemies(transform.position, 0.15f, HealAmount);
+        Invoke("AoEHealWithInterval", HealInterval);
+    }
+
+    private void HealEnemies(Vector3 center, float radius, float amount)
+    {
+        var enemiesInRange = Physics.OverlapSphere(center, radius);
+
+        if (enemiesInRange.Length < 1) return;
```

Comment:
Not necessary to return since the while loop body won't be reached when `enemiesInRange` is < 1